runtime-builder/tests/integration/go-get.sh

```diff
@@ -8,35 +8,23 @@ new_gopath=$(pwd -P)
 echo "Cleaning up dependencies..."
```

Comment:
can you use `set -euo pipefail` at the top of this script? `set -e` won't catch all failure cases